prow/config/jobs/metal3-io/ironic-standalone-operator-release-0.6.yaml

```
@@ -0,0 +1,66 @@
+presubmits:
+  metal3-io/ironic-standalone-operator:
+  - name: gomod
+    branches:
+    - release-0.6
+    skip_if_only_changed: '(((^|/)OWNERS)|((^|/)OWNERS_ALIASES)|(\.md))$'
+    decorate: true
+    spec:
+      containers:
+      - args:
+        - ./hack/gomod.sh
+        command:
+        - sh
+        env:
+        - name: IS_CONTAINER
+          value: "TRUE"
+        image: docker.io/golang:1.24
+        imagePullPolicy: Always
+  - name: markdownlint
+    branches:
+    - release-0.6
+    run_if_changed: '(\.md|markdownlint\.sh)$'
+    decorate: true
+    spec:
+      containers:
+      - args:
+        - ./hack/markdownlint.sh
+        command:
+        - sh
+        env:
+        - name: IS_CONTAINER
+          value: "TRUE"
+        image: docker.io/pipelinecomponents/markdownlint-cli2:0.12.0@sha256:a3977fba9814f10d33a1d69ae607dc808e7a6470b2ba03e84c17193c0791aac0
+        imagePullPolicy: Always
+  - name: manifestlint
+    branches:
+    - release-0.6
+    skip_if_only_changed: '(((^|/)OWNERS)|((^|/)OWNERS_ALIASES)|(\.md))$'
+    decorate: true
+    spec:
+      containers:
+      - args:
+        - ./hack/manifestlint.sh
+        command:
+        - sh
+        env:
+        - name: IS_CONTAINER
+          value: "TRUE"
+        - name: KUBECONFORM_PATH
+          value: "/"
+        image: ghcr.io/yannh/kubeconform:v0.6.7-alpine@sha256:824e0c248809e4b2da2a768b16b107cf17ada88a89ec6aa6050e566ba93ebbc6
+        imagePullPolicy: Always
+  #NOTE(elfosardo): commented out until metal3-dev-env starts using IrSO
+  # name: metal3-dev-env-integration-test-{image_os}-release-1-10
```

```suggestion
  # name: metal3-dev-env-integration-test-{image_os}-release-1-11
```

Maybe update the 1.10 references to 1.11